### DISCUSSION TITLE
DAG Import Errors

### DISCUSSION THREAD

**Discussion by chanurholis** [apache/airflow#24803] [Q&A]
![image](https://user-images.githubusercontent.com/55186337/177008755-9808d2e2-7c16-404a-8b09-5ac14994547b.png)

I came across this error when I started running airflow using docker.

I've tried adding :
`airflow-worker:
     extra_hosts: 
       - "host.docker.internal:host-gateway"`

**Comment by potiuk** [comment]
Yes, You ave a problem with resolving that hosr name. 

You might find this one insightful
https://stackoverflow.com/questions/48546124/what-is-linux-equivalent-of-host-docker-internal

**Reply by chanurholis** [reply]
hi @potiuk 

thanks for responding, i will try 👍🏻
